tests/emscripten_log/emscripten_log.cpp

```diff
@@ -105,23 +105,6 @@ void __attribute__((noinline)) bar(int = 0, char * = 0, double = 0) // Arbitrary
 	MYASSERT(!!strstr(buffer, "at bar(int,"), "Truncated callstack was %s!", buffer);
 	MYASSERT(buffer[20] == 0x01, "");
 	delete[] buffer;
-
-	/* With EM_LOG_JS_STACK, the callstack will be
-		at __Z3bariPcd (src.cpp.o.js:5394:12)
-		at __Z3FooIiEvv (src.cpp.o.js:5417:4)
-		at Object._main (src.cpp.o.js:5404:2)
-		at Object.callMain (src.cpp.o.js:71344:30)
-		at doRun (src.cpp.o.js:71383:25)
-		at run (src.cpp.o.js:71396:5)
-		at Object.<anonymous> (src.cpp.o.js:71439:1)
-		at Module._compile (module.js:456:26) */
-#ifdef RUN_FROM_JS_SHELL
-	MYASSERT(!!strstr(str, "at __Z3bariPcd (src.cpp"), "Callstack was %s!", str);
-	MYASSERT(!!strstr(str, "at __Z3FooIiEvv (src.cpp"), "Callstack was %s!", str);
-#else
-	MYASSERT(!!strstr(str, "at __Z3bariPcd (page.js"), "Callstack was %s!", str);
-	MYASSERT(!!strstr(str, "at __Z3FooIiEvv (page.js"), "Callstack was %s!", str);
-#endif
```

Comment:
do we not want to eventually fix this so it passes (so a TODO instead of deleting)? and why does this part not pass?

Reply:
No, I don't think there is any value in this part of the test.   I think this part of the tests was written for the fastcomp days.    I'll need to revisit to see exactly why I think that, but basically the backtrace always has demangled symbols in wasm mode... there is no way to get these mangled symbols anymore as far as I know.. because we always put demangled names in the names section.. or none at all.

Reply:
I see, sounds good.